[website] Open 4 new roles (#27123)
mui-org/material-ui: docs/src/pages/company/careers/full-stack-engineer.md

# Full-stack Engineer

<p class="description">We are looking for a full-stack engineer to pioneer the development of a new product.</p>

## Details of the Role

- Location: Remote (preference for UTC-5 to UTC+4).
- Type of work: Contractor (full-time, convertible to an employee position in the future).
- Start date: Immediately.
- Level: [4 or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).
- We're a remote company, we prefer asynchronous communication over meetings.
- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.

## About us

See the [careers](/company/careers/) and [about us](https://next.material-ui.com/branding/about/) pages.

### Why we're hiring

Both our open-source community and our premium products are growing fast (x2-3 YoY).
We need talented people to keep that going!

Our mission is to make building UIs more accessible.

> The faster, simpler, and the broader the audience that can create custom UIs, the better.

Reviewer: ```suggestion
The faster and simpler it is, and the broader the audience that can create custom UIs, the better.
```

> We believe that the best way to improve on these dimensions is to reduce [80%](https://www.youtube.com/watch?v=GnO7D5UaDig&t=2451s) of the code that has to be written.

Reviewer: ```suggestion
We believe that the best way to improve on these dimensions is to eliminate [80%](https://www.youtube.com/watch?v=GnO7D5UaDig&t=2451s) of the code that has to be written.
```

### Why this is interesting

> Our solution is empowering React developers to build awesome applications. Hundreds of thousands of developers use Material-UI every month.

Reviewer: ```suggestion
Our solution empowers React developers to build awesome applications – hundreds of thousands of developers use Material-UI every month.
```

But providing React components isn't enough.

> In our [last developers' survey](/blog/2020-developer-survey-results/), we have learned that the majority of our audience are full-stack developers, they are looking for ways to move faster, working on a couple of new projects every year, and where the integration with data is key.

Reviewer: ```suggestion
In our [last developer survey](/blog/2020-developer-survey-results/), we learned that the majority of our audience are full-stack developers. They are looking for ways to move faster, working on a couple of new projects every year, and where the integration with data is key.
```

## About the role

You will initiate the development of a bold new product vertical. We are looking for an experienced, and ambitious full-stack engineer that is ready to work in an entrepreneurial environment.

### What you'll do on a day-to-day basis

- **Define the roadmap** and refine the product direction.
- **Take ownership of features from idea/mockup to live deployment**. You'll shape user-facing features with everything from database models to UI components.
- **Ship. Early and often**. You'll iterate and ship frequently. You'll have a real impact on the end-user experience and you'll love working on a team that builds stunning UIs and prioritizes delivering real user value as often as possible.
- **You'll be interacting with the early users** on a regular basis, handling inbound support and feature requests.

## About you

You are a [manager of one](https://signalvnoise.com/posts/1430-hire-managers-of-one).
You are curious, you enjoy taking risks, and learning.

### Skills you should have

> - **Expertise in the modern JavaScript ecosystem**. Material-UI is built on the shoulders of giants, making use of technologies such as ES2015+, TypeScript, Node.js, React, Next.js, webpack, and Babel.

Reviewer: But no AWS Infinidash... 😢

- ​You are autonomous and a faster learner. You are proactive and can start projects without constant direction.
- You ship code often that is elegant to use and read, and you take ownership of it. You can be relied upon throughout its lifecycle.
- You can put a lot of thoughts into design, product roadmap, and the details of each feature. You won't only take a project and make it "functional."
- You communicate your thoughts and decisions clearly and proactively, and engage with the rest of the team with an optimistic and first principles mindset.

### What would be nice if you had, but isn't required

- You've maintained an active repository before. Maybe you've helped maintain a popular open-source repo, or perhaps you've worked on internal repos that saw contributions from multiple teams.

## Compensation

> A competitive compensation of up to \$140,000 USD/year, depending on the profile. It could go higher for a person that can significantly push the mission forward. You can find the other perks & benefits in the [careers](/company/careers/#perks-amp-benefits) page.

Reviewer: ```suggestion
Competitive compensation of up to \$140,000 USD/year, depending on the profile. It could go higher for a person that can significantly push the mission forward. You can find the other perks & benefits in the [careers](/company/careers/#perks-amp-benefits) page.
```
